lib/jsdom/living/nodes/HTMLOutputElement-impl.js

```diff
@@ -19,6 +20,10 @@ class HTMLOutputElementImpl extends HTMLElementImpl {
   get labels() {
     return getLabelsForLabelable(this);
   }
+
+  get form() {
+    return closest(this, "form");
```

Comment:
This does not fully implement https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#form-owner . Does implementing it in this incomplete way help pass other tests? (E.g. shadow DOM tests.) If so then this is OK, but add a TODO comment explaining that it's incomplete.

If it's not necessary for shadow DOM tests, then let's remove this change until we can do a complete implementation.